fedmsg_meta_fedora_infrastructure/anitya.py

```diff
@@ -124,24 +124,33 @@ def subtitle(self, msg, **config):
             else:
                 message = msg['msg']
 
-            # If this project is mapped to a Fedora package, then just
-            # substitute "our" package name for the project name to make
+            # If this project is mapped to one or more Fedora package, 
+            # then just add the mapped packages to make
             # everyone's emails more readable.
             # https://github.com/fedora-infra/the-new-hotness/issues/21
+            packages = []
             for package in message.get('packages', []):
                 if package['distro'] == 'Fedora':
-                    project = package['package_name']
-
+                    packages.append(package['package_name'])
+            
+            if len(packages) == 1:
+                packages = packages[0]
+                text = "package"
+            else:
+                packages = " and ".join([", ".join(packages[0:-1]),packages[-1]])
```

Comment:
@ralphbean,  we can use that function,  not a bad idea, but for `text = "package"`/`text="packages"` we will end up using the `if..else..`.  So what i am saying is if we use `list_to_series` we can avoid the `if...else...` but then the `text` will be fixed (`package`) for single or multiple packages. Otherwise if we are going to use `if..else..` for deciding the `text` then its best as it is, calling `list_to_series' will be overhead.